src/NeuralNet/CostFunctions/CrossEntropy/CrossEntropy.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rubix\ML\NeuralNet\CostFunctions\CrossEntropy;
```

Comment:
Are you are carving out this namespace to hold the new classes temporarily until we can get the whole Neural Net subsystem converted to NumPower?